terraform/production/main.tf

```diff
@@ -0,0 +1,25 @@
+terraform {
+  backend "s3" {
+    bucket  = "terraform-state-production-apis"
+    encrypt = true
+    region  = "eu-west-2"
+    key     = "services/hackney-playbook/state"
+  }
+}
+
+provider "aws" {
+  region = "us-east-1"
```

Comment:
Shouldn't this be eu-west-2?

Reply:
No, because it's cloudfront it needs to be us-east-1.

See documentation [here](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/edge-functions-restrictions.html#:~:text=The%20Lambda%20function%20must%20be%20in%20the%20US%20East%20(N.%20Virginia)%20Region.).